src/test/java/seedu/anilist/logic/commands/CommandTestUtil.java

```diff
@@ -31,50 +30,128 @@
  */
 public class CommandTestUtil {
 
+    // Invalid Strings
+    public static final String INVALID_STRING_SPACE = " ";
+    public static final String INVALID_STRING_EMPTY = "";
+    public static final String INVALID_STRING_NON_ASCII = "中文";
+
+
+    //-----------------------------------------------------NAME---------------------------------------------------------
+    // Valid Names
     public static final String VALID_NAME_AKIRA = "Akira";
     public static final String VALID_NAME_BNHA = "Boku No Hero Academia";
+    public static final String VALID_NAME_EIGHTY_SIX = "86";
+    public static final String VALID_NAME_FATE_ZERO = "Fate/Zero";
+    public static final String VALID_NAME_JOJO_GOLDEN_WIND = "JoJo's Bizarre Adventure: Golden Wind";
+    public static final String VALID_NAME_NON_ALPHANUMERIC_ASCII = "~!@#$%^&()+-=:;";
     public static final String VALID_NAME_MAX_LENGTH = "A".repeat(140);
-    public static final String VALID_GENRE_ACTION = "action";
-    public static final String VALID_GENRE_SCIENCE_FICTION = "sci fi";
-    public static final String VALID_EPISODE_ONE = "1";
-    public static final String VALID_EPISODE_TWO = "2";
-    public static final String VALID_ACTION_ADD = "add";
-    public static final String VALID_ACTION_DELETE_SHORT_FORM = "d";
-    public static final String VALID_STATUS_TOWATCH = "towatch";
-    public static final String VALID_STATUS_WATCHING = "watching";
 
-    public static final String INVALID_ACTION_EMPTY_STRING = "";
-    public static final String INVALID_EPISODE_LARGER_THAN_MAX_INT =
-        Long.toString((long) Integer.MAX_VALUE + 1);
-    public static final String INVALID_ACTION_NO_SUCH_ACTION = "dancing";
-    public static final String INVALID_GENRE_NOT_IN_LIST = "tentacles";
+    // Invalid Names
+    public static final String INVALID_NAME_LONGER_THAN_MAX_LENGTH = "F".repeat(142);
```

Comment:
the max length here you can probably import a constant from the Name class

Reply:
ok